aries-site/src/pages/components/accordion.js

```diff
@@ -18,23 +19,72 @@ const Accordion = () => (
     />
     <ContentSection>
       <Subsection name={title} level={1} topic={topic}>
-        <SubsectionText>
-          The accordion affords content to be delivered progressively. The
-          chevron is used to identify the expand or collapse action while the
-          entire header can be clicked to expand or collapse content.
-        </SubsectionText>
+        <SubsectionText>{page.description}</SubsectionText>
         <Example
-          details={[
-            `When seeking to provide maximum content in limited, vertical 
-            space, an accordion is a good alternative.`,
-          ]}
           docs="https://v2.grommet.io/accordion#props"
           code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/components/controls/AccordionExample.js"
+          figma="https://www.figma.com/file/UvTBUQdhk07wV79nijsrtr/HPE-Accordion-Component?node-id=71%3A25"
         >
           <AccordionExample />
         </Example>
       </Subsection>
     </ContentSection>
+    <ContentSection>
+      <Subsection name="Guidance">
+        <SubsectionText>
+          When seeking to provide maximum content in limited, vertical space, an
+          accordion is a good alternative.
+        </SubsectionText>
+        <SubsectionText>
+          An accordion is a group of vertically stacked collapsible buttons with
+          title headers. This gives the user a high level idea of what content
+          will be disclosed before collapsing a section.
+        </SubsectionText>
+        <SubsectionText>
+          Accordions make it easy for the user to process and find the
+          information they need in a more effective manner. Accordions can be
+          used to organize information, which helps the user consume the
+          information they need at a faster pace.
+        </SubsectionText>
+      </Subsection>
+      <Subsection name="About Accordion" level={3} gap="small">
+        <SubsectionText>
+          Accordion has two states:
+          <BulletedList level={3} items={['Collapsed', 'Expanded']} />
+          The chevron is used to identify the expand or collapse action while
```

Comment:
```suggestion
          The chevron icon is used to identify the expand or collapse action while
```
Or, I'm wondering if we use a more common term we could use instead of chevron

Reply:
yeah this part was already there I can change to just icon